src/platform/Linux/CHIPDevicePlatformEvent.h

```diff
@@ -45,6 +45,8 @@ enum PublicPlatformSpecificEventTypes
 enum InternalPlatformSpecificEventTypes
 {
     kPlatformLinuxEvent = kRange_InternalPlatformSpecific,
+    kPlatformLinuxBLEAdapterAdded,
```

Comment:
could you replace the enum with the enum class, then specify uint8_t?
this way, we can reduce the scope leak here. thanks

Reply:
I've been thinking about this but as a follow-up. There are more enums which could be changed to `enum class`. Also, for consistency this should be done in all platforms (as of now all platforms have `enum InternalPlatformSpecificEventTypes`). What do you think about another PR instead of adding complexity to this one?

Reply:
sure, could you file a ticket on it for follow-up? thanks

Reply:
Issue for follow-up PR: https://github.com/project-chip/connectedhomeip/issues/32755